index.rst

```diff
@@ -100,6 +95,12 @@ Additionally, EditorConfig defines the following terms:
 - Section Name: the string between the beginning ``[`` and the ending ``]``.
 - Section: the lines starting from a Section Header until the beginning of
   the next Section Header or the end of the file.
+ 
+ 
+   The EditorConfig file format formerly allowed the use of `;` and `#` after the
+   beginning of the line to mark the rest of a line as comment. This led to
+   confusion how to parse values containing those characters. Old EditorConfig
+   parsers may still allow inline comments.
```

Comment:
The leading whitespace would probably mess up the formatting...

Technically we didn't formerly "allowed" inline comment, but we have left them undefined. So the best wording might be "formerly left the use of `;` and `#` after the beginning of the line to mark the rest of a line as comment undefined"?